extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1200,6 +1242,23 @@ static public function IDtoDB( $city_id, $master = false ) {
 		return isset( $oRow->city_dbname ) ? $oRow->city_dbname : false;
 	}
 
+	/**
+	 * "Unlocalizes" the host replaces env-specific domains with "wikia.com", for example
+	 * 'muppet.preview.wikia.com' -> 'muppet.wikia.com'
+	 *
+	 * @param $host
+	 * @return string normalized host name
+	 */
+	protected static function normalizeHost( $host ) {
+		global $wgDevDomain;
+		// strip env-specific pre- and suffixes for staging environment
+		$host = preg_replace( '/\.(stable|preview|verify|sandbox-[a-z0-9]+)\.wikia\.com/', static::WIKIA_TOP_DOMAIN, $host );
```

Comment:
Can we use `$wgWikiaBaseDomain` in place of the hardcoded `wikia\.com` while we're touching this code? i.e. `preg_quote( $wgWikiaBaseDomain )`.

Reply:
good point, done